### PR TITLE
Fix command line build to work with azure pipelines

### PR DESCRIPTION
**Pull request by MenelvagorMilsom**
Overview
---
For some reason, calls to StartProcessAsync were hanging on Azure Pipelines CI, even though the processes were running properly and there were task state transitions to `RanToCompletion`. In order to unblock CI, I've tentatively replaced them with synchronous Process.WaitForExit.

### CODE REVIEW

File: Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/UwpAppxBuildTools.cs [hunk 1]
```diff
@@ -170,41 +170,56 @@ public static async Task<bool> BuildAppxAsync(string productName, bool forceRebu
...
             {
-                Debug.LogError($"{PlayerSettings.productName} appx build Failed! (ErrorCode: {processResult.ExitCode})");
+                process.StartInfo = new ProcessStartInfo
+                {
+                    FileName = msBuildPath,
+                    CreateNoWindow = true,
+                    UseShellExecute = false,
+                    RedirectStandardOutput = false,
+                    RedirectStandardError = false,
+                    Arguments = $"\"{solutionProjectPath}\" /t:{(forceRebuildAppx ? "Rebuild" : "Build")} /p:Configuration={buildConfig} /p:Platform={buildPlatform} /verbosity:m -fl -flp:logfile=appxbuild.log;verbosity=diagnostic",
```
Comment (MenelvagorMilsom) [inline Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/UwpAppxBuildTools.cs]:
```suggestion
                    Arguments = $"\"{solutionProjectPath}\" /t:{(forceRebuildAppx ? "Rebuild" : "Build")} /p:Configuration={buildConfig} /p:Platform={buildPlatform} /verbosity:m",
```
Left over from testing

**Comment by MenelvagorMilsom** [comment]
Sounds good, most of these changes were simply to unblock CI in the short term. If you have more changes and improvements that don't break CI, I can think of no reason why we wouldn't favour those changes once they're ready.

**Comment by StephenHodgson** [comment]
Yup I had to make similar changes to get my forks CI working as well.

**Comment by StephenHodgson** [comment]
@MenelvagorMilsom opening a PR in a few minutes. Just doing some final checks

**Comment by StephenHodgson** [comment]
@davidkline-ms please do not merge this PR, I'd like to fix this issue with the changes I ported in #3618.

**Comment by MenelvagorMilsom** [comment]
The issues addressed by this PR are fixed with #3618

**Pull request closed without merging**